misc/python/materialize/parallel_workload/data_type.py

```diff
@@ -81,7 +81,29 @@ def name() -> str:
     @staticmethod
     def value(rng: random.Random, in_query: bool = False) -> Any:
         result = rng.randint(-100, 100)
-        return f"'{result}'" if in_query else result
+        return f"'{result}'" if in_query else str(result)
+
+
+class Jsonb(DataType):
+    @staticmethod
+    def name() -> str:
+        return "jsonb"
+
+    @staticmethod
+    def value(rng: random.Random, in_query: bool = False) -> Any:
+        result = rng.randint(-100, 100)
+        return f"'{result}'::jsonb"
```

Comment:
Is this a valid JSON string?

Reply:
It's a very simple json string, but yes, numbers are valid json:
```
deen=> select '100'::jsonb;
 jsonb
-------
 100
(1 row)
```